enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -79,4 +84,62 @@ <h3 class="course-confirmation-title">{{ confirmation_text }}</h3>
       </div>
     </div>
   </div>
+  <div id="course-details-modal" class="modal" aria-hidden="true">
+    <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+      <button class="modal-close-button" id="modal-close-button" aria-label="{{ close_modal_button_text }}">{{ close_modal_button_text }}</button>
+      <header class="wrapper">
+        <div class="image">
+          <img src="{{ course_image_uri }}" alt="{{ course_name }}"/>
+        </div>
+        <div class="details">
+          <h1 id="modal-header-text">{{ course_name }}</h1>
+          <p class="short-description">{{ course_short_description }}</p>
+          <div class="organization">
+            <img src="{{ organization_logo }}" alt="{{ organization_name }}" />
+          </div>
+        </div>
+      </header>
+      <div class="body">
+        <div class="details">
+          <ol>
+            <li>
+              <span class="icon fa fa-tag" aria-hidden="true"></span>
+              <span class="title">{{ price_text }}</span>
+              <span class="text">
+                {% if course_modes|length > 0 %}
+                  {% with course_mode=course_modes.0 %}
+                    {% if course_mode.final_price and course_mode.original_price != course_mode.final_price %}
+                      <strike>{{ course_mode.original_price }}</strike>
+                      <br />
+                      <span class="discount">{{ course_mode.final_price }}</span>
+                    {% else %}
+                      {{ course_mode.original_price }}
```

Comment:
@chris-mike do you have any input on what we should do if there are no course modes for the course? This affects the price we display. There are also similar questions below around what to do if the other information around level and effort are missing in the metadata.